Frameworks/CoreImage/CIContext.mm

```diff
@@ -74,8 +74,7 @@ + (CIContext*)contextWithEAGLContext:(EAGLContext*)eaglContext options:(NSDictio
  @Notes parameter dict is ignored.
 */
 + (CIContext*)contextWithOptions:(NSDictionary*)dict {
-    CIContext* context = [[CIContext alloc] init];
-    return context;
+    return [[CIContext alloc] init];
 }
```

Comment:
(I couldn't determine whether ARC was on for this framework.)

---
In reply to: [94636333](https://github.com/Microsoft/WinObjC/pull/1625#discussion_r94636333) [](ancestors = 94636333)